index.html

```diff
@@ -10,8 +10,119 @@
+            <div class="col-mobile-100">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati perferendis quos ab quam quia? Vitae, ad facere veritatis expedita ducimus laudantium iste aut ipsa suscipit, ullam quasi eos voluptatem quia?</p>
+            </div>
+        </div>
+        <!-- TODO: fix layout -->
+        <div class="three-columns shop row">
+            <div class="col-mobile-100 col-large-tablet-33">
+                <div class="col-large-tablet-50">
+                    <p>lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati perferendis quos ab quam quia?</p>
+                </div>
+                <div class="col-large-tablet-50">
+                    <p>lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati perferendis quos ab quam quia?</p>
+                </div>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati perferendis quos ab quam quia? Vitae, ad facere veritatis expedita ducimus laudantium iste aut ipsa suscipit, ullam quasi eos voluptatem quia?</p>
+            </div>
+            <div class="col-mobile-100 col-large-tablet-33">
+                <div class="col-large-tablet-50">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita voluptatem architecto et hic repellat aut molestias, incidunt officia possimus ducimus qui! Voluptas sint sunt eaque consectetur facere repellat, minus enim!</p>
+                </div>
+                <div class="col-large-tablet-50">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita voluptatem architecto et hic repellat aut molestias, incidunt officia possimus ducimus qui! Voluptas sint sunt eaque consectetur facere repellat, minus enim!</p>
+                </div>
+            </div>
+            <div class="col-mobile-100 col-large-tablet-33">
+                <div class="col-large-tablet-100">
+                    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Iusto minus sunt, soluta facere, quidem deleniti possimus repellat ab id doloremque illum voluptas? Nemo, nulla. Laborum doloribus earum maxime. Voluptatum, ipsum!</p>
+                </div>
+                <div class="col-large-tablet-100">
+                    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Iusto minus sunt, soluta facere, quidem deleniti possimus repellat ab id doloremque illum voluptas? Nemo, nulla. Laborum doloribus earum maxime. Voluptatum, ipsum!</p>
+                </div>
+            </div>
+        </div>
+        <div class="cta row">
+            <div class="col-mobile-100">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati perferendis quos ab quam quia? Vitae, ad facere veritatis expedita ducimus laudantium iste aut ipsa suscipit, ullam quasi eos voluptatem quia?</p>
+            </div>
+        </div>
+    </main>
 
+    <footer class="container-fluid">
```

Comment:
![image](https://user-images.githubusercontent.com/38762883/84188599-c590d200-aa8b-11ea-80f9-6a0d2cd76cb0.png)
Tutaj bd potrzebna zmiana bo bd mial problem z ostylowaniem kolumn footera. Ja bym to zmienil tak:
1.) Usunal byl klase container-fluid ze znacznika footer
2.) Wewnatrz znacznika footer utworzylbym 2 divy dla:
    - newsletter (ten powinien byc container-fluid)
    - footer-bottom (ten powinien byc container)
Dzieki temu latwiej ci bedzie ostylowac footer bottom (zauwaz ze jego kolumny nie sa fluid)